content/docs/other-guides/istio-in-kubeflow.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Istio usage in Kubeflow"
+description = "Managing access to Kubeflow applications and resources via Istio"
+weight = 50
++++
+
+Kubeflow 0.6 onwards deploys Istio along with configuration to enable
```

Comment:
In the docs, we add a "v" in front of the version number in all cases. So, "v0.6". The reason is to be consistent in all usages, because the version tag that people have to use when the download kfctl does include the "v".

Reply:
done